buildchain/static-container-registry/test.sh

```diff
@@ -21,31 +21,31 @@ TEST_SKOPEO=${TEST_SKOPEO:-1}
 
 if [ "$TEST_DOCKER" -eq 1 ]; then
 test_docker() {
-        for image in ${AVAILABLE_IMAGES[*]}; do
+        for image in "${AVAILABLE_IMAGES[@]}"; do
```

Comment:
Shouldn't we rather ignore these files when running shellcheck? They come from another repository (which I don't remember how we imported though), we shouldn't really care IMO.

Reply:
Indeed, maybe we should